lib/actions/sessions.js

```diff
@@ -88,7 +88,7 @@ function createExitAction(type) {
         }
 
         const sessions = keys(getState().sessions.sessions);
-        if (!sessions.length) {
+        if (sessions.length === 0) {
```

Comment:
`!something` and `!something.length` is not the same thing. The first one is a lot clearer than the latter.